packages/framework-core/src/booster.ts

```diff
@@ -101,6 +104,19 @@ export class Booster {
   ): Promise<TEntity | undefined> {
     return fetchEntitySnapshot(this.config, this.logger, entityClass, entityID)
   }
+
+  /**
+   * Destroy all the entity events and snapshots. That action will cause the entity deletion, since
+   * we'll be not able to reduce its state based on its events and snapshots.
+   * @param entityName Name of the entity class
```

Comment:
The argument is called `entityClass`, I don't know if you wanted to call it `entityName` or if this is a typo ⬇️ 
```suggestion
   * @param entityClass Name of the entity class
```